requirements_tools/upgrade_requirements.py

```diff
@@ -108,7 +108,7 @@ def pip_install(pip, *argv):
             print_call(pip, 'install', '-i', args.index_url, *argv)
 
         # Latest pip installs python3.5 wheels
-        pip_install(pip, 'pip', 'setuptools', args.pip_tool, '--upgrade')
+        pip_install(pip, 'setuptools', args.install_deps, '--upgrade')
         pip_install(pip_tool, '-r', 'requirements-minimal.txt')
```

Comment:
I *think* `pip_tool` needs to be `shlex.split`d since it could be a set of arguments (see how check-all-wheels does this)

Reply:
yeah that's out of scope, but `pip_tool` it does